ExternalModule.php

```diff
@@ -71,7 +102,7 @@ function sendEmail($user_info) {
 		$sender = $project_contact_email ?? 'CTSI-REDCAP-SUPPORT-L@lists.ufl.edu';
 		$subject = $this->getSystemSetting("wups_subject");
 		$body = $this->getSystemSetting("wups_body");
-		$activation_link = $this->getUrl('activate_account.php'). "&username=" . $user_info['username'];
+		$activation_link = APP_PATH_WEBROOT_FULL . "?username=" . $user_info['username'];
```

Comment:
You don't get to own the `username` option. How about instead you say something like

`$activation_link = APP_PATH_WEBROOT_FULL . "?reset_suspension_time&username=" . $user_info['username'];`

You'll also need to change line 23 above to match.